src/ripple/app/tx/impl/SetHook.cpp

```diff
@@ -937,7 +943,7 @@ SetHook::destroyNamespace(
         sleAccount->setFieldU32(sfHookStateCount, stateCount);
 
     if (ctx.rules.enabled(fixNSDelete))
-        adjustOwnerCount(view, sleAccount, -toDelete.size(), ctx.j);
+        adjustOwnerCount(view, sleAccount, -toDeleteStateCount * scale, ctx.j);
```

Comment:
we should do a sanity check on this value before passing it to make sure it is reasonable and hasn't overflowed

Reply:
https://github.com/Xahau/xahaud/commit/a75daaea0249c1fabd773c4ef1ee08929cd9d3a3